### PR TITLE
Sorted movies by rating for a given year

### PR DESCRIPTION
**Pull request by sharpchris**
Day 6 project show movies by a given year, sorted by rating

### CODE REVIEW

File: days/04-06-collections/chris/movies_by_year.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+years_of_movies = defaultdict(list)
+
+with open('movie_metadata.csv', encoding='utf-8') as f:
+    movies_csv = csv.DictReader(f)
+    for row in movies_csv:
+        movie = Movie(title=row['movie_title'],rating=row['imdb_score'])
+        years_of_movies[row['title_year']].append(movie)
+    
+def get_movies_by_year(year):
+    print(f'Here are all the movies released in {year} sorted by rating:',end='\n\n')
```
Comment (bryanfctobin) [inline days/04-06-collections/chris/movies_by_year.py:18]: Getting the below error originating from line 18.  Could be me, could we take a look today before I merge this?

`Bryan-Tobin's-MacBook-Pro:folder bryantobin$ python test.py
  File "test.py", line 18
    print('Here are all the movies released in {year} sorted by rating:',end='\n\n')
                                                                            ^
SyntaxError: invalid syntax`

**Review by bryanfctobin — CHANGES_REQUESTED** [review]
Getting the following error from line 18:
`Bryan-Tobin's-MacBook-Pro:folder bryantobin$ python test.py
  File "test.py", line 18
    print('Here are all the movies released in {year} sorted by rating:',end='\n\n')
                                                                            ^
SyntaxError: invalid syntax`

File: days/04-06-collections/chris/movies_by_year.py (hunk 1 above)
Comment (sharpchris) [reply days/04-06-collections/chris/movies_by_year.py:18]: It looks like your print is missing the "f" that would precede the first quotation mark after the opening parentheses. What version of Python are you using?

**Comment by bryanfctobin** [comment]
Looks good, and there were a ton of cool movies that were released in 1988

**Pull request merged**